reference/array/functions/array-is-list.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- $Revision$ -->
+<!-- $Author: haytoo $ -->
+<!-- EN-Revision: 5ecd5e8e50f0f47dce7341ddc95334af62d9e770 Maintainer: haytoo Status: ready -->
+<refentry xml:id="function.array-is-list" xmlns="http://docbook.org/ns/docbook">
+ <refnamediv>
+  <refname>array_is_list</refname>
+  <refpurpose>Checks whether a given <parameter>array</parameter> is a list</refpurpose>
+ </refnamediv>
+ 
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <type>bool</type><methodname>array_is_list</methodname>
+   <methodparam><type>array</type><parameter>array</parameter></methodparam>
+  </methodsynopsis>
+  <para>
+   判断指定的数组是否是个 list。如果数组的 key 由 <literal>0</literal> 到 <literal>count($array)-1</literal>组成，则该数组就是 list。
+  </para>
+ </refsect1>
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  <para>
+   <variablelist>
+    <varlistentry>
+     <term><parameter>array</parameter></term>
+     <listitem>
+      <para>
+       被检测的数组
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </para>
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   如果数组是 list 就返回 true，否则返回 false。
```

Comment:
`Returns &true; if <parameter>array</parameter> is a list, &false;
   otherwise.`

这里的 `&true;` 和  `&false;` 和 `<parameter>array</parameter>` 同理，维持不变。